Arm/Hand/hand.hpp

```diff
@@ -27,52 +27,101 @@ class Hand
 
   Hand(sjsu::arm::WristJoint wrist) : wrist_(wrist) {}
```

Comment:
We need to pass by reference here still 
```suggestion
  Hand(sjsu::arm::WristJoint & wrist) : wrist_(wrist) {}
```